providers/types.yaml

```diff
@@ -86,7 +86,7 @@ components:
       properties:
         grantType:
           type: string
-          enum: [authorizationCode, clientCredentials, PKCE]
+          enum: [authorizationCode, clientCredentials, password, PKCE]
```

Comment:
we're starting to migrate all of our openapi definition to its own repo, can you make the same change in this file also? https://github.com/amp-labs/openapi/blob/main/catalog/catalog.yaml

Reply:
PR created: https://github.com/amp-labs/openapi/pull/4